test/functional/wallet_taproot.py

```diff
@@ -5,6 +5,7 @@
 """Test generation and spending of P2TR addresses."""
 
 import random
+import uuid
```

Comment:
This triggers a segfault on my Ubuntu 22.10 machine (Python 3.6.15 installed via PyEnv, Kernel 6.0.6). I'm able to [work around](https://github.com/pyenv/pyenv/issues/2239#issuecomment-1079275184) it:

```
CC=clang pyenv install 3.6.15
```